### DISCUSSION TITLE
Example Nav Drawer (offcanvas)

### DISCUSSION THREAD

**Discussion by Dezmonter** [themesberg/flowbite#230] [General]
Thanks for the great library! Tell me, will you have an example of creating a nav using Drawer (offcanvas) ?

**Comment by zoltanszogyenyi** [comment]
Hello @Dezmonter,

Thanks for the warm thoughts!

We actually do have a [drawer navigation](https://flowbite.com/docs/components/drawer/#drawer-navigation) which is kind of a sidebar.

Or did you mean something else?

PS: moved this to the discussions page.
